data/txts/developers.json

```diff
@@ -373,6 +373,24 @@
 				}
 			]
 		},
+		{
+			"heading": "Add-Ons",
+			"image": "images/players/genstats_player.png",
```

Comment:
How about using an image with a scroll (for Scripting) here? E.g. like for the homepage

Reply:
Something like this?

![addon_authors](https://user-images.githubusercontent.com/6730556/119689318-0888b880-be49-11eb-8562-2f82fa03e239.png)

Reply:
Or bronwish:

![addon_authors_brown](https://user-images.githubusercontent.com/6730556/119690522-0f63fb00-be4a-11eb-8429-5ec7193fad5d.png)

I think this is better.

Reply:
I like the first one better. Perhaps place the image in `data/images/ui_fsmenu` since it'll be displayed in the FsMenu's About window.

Reply:
Created a new folder add_ons… maybe we need some more add-on specific images.

Reply:
Sorry didn't saw your post. I have to wait for OIPUN anyway and make the changes then.